src/main/java/org/openstreetmap/atlas/checks/distributed/AtlasDataSource.java

```diff
@@ -193,6 +193,6 @@ private Atlas loadPbf(final Resource input, final String country)
         final CountryBoundaryMap map = CountryBoundaryMap
                 .fromBoundaryMap(Collections.singletonMap(country, this.polygon));
         final AtlasLoadingOption option = AtlasLoadingOption.createOptionWithAllEnabled(map);
-        return new OsmPbfLoader(input, this.polygon, option).read();
+        return new RawAtlasGenerator(input, option, this.polygon).build();
```

Comment:
Sorry, chiming in late - is this intended to be country-sliced and way-sectioned? In that case, you may want to add the following code: 

```
atlas = new RawAtlasGenerator(input, option, this.polygon).build();
if (option.isCountrySlicing())
{
       atlas = new RawAtlasCountrySlicer(
       option.getCountryBoundaryMap().getLoadedCountries(),
       option.getCountryBoundaryMap()).slice(atlas);
}
if (option.isWaySectioning())
{
     atlas = new WaySectionProcessor(atlas, option).run();
}
```
Otherwise, you're only creating a raw atlas (which has points, lines, relations) and nothing else (not sliced, sectioned, doesn't have edges, areas or nodes). cc. @jklamer

Reply:
Ah this is my bad. @jklamer had me review this and I said it looked OK. I should have thought more carefully about what was going on.